docs/cli/built-in-commands/addons.md

```diff
@@ -0,0 +1,68 @@
+# Add-on Management
+
+Set of CLI commands that allows listing, installing, updating and uninstalling add-ons.
+
+[TOC]
+
+## `addons:list`
+
+Lists all add-ons available in the system.
+
+    php eecli.php addons:list
+
+List add-ons that can be installed:
+
+    php eecli.php addons:list i
+
+or
+
+    php eecli.php addons:list installed
+
+List add-ons that have an update available:
+
+    php eecli.php addons:list a
+
+or
+
+    php eecli.php addons:list update-available
+
+List add-ons that can be uninstalled:
+
+    php eecli.php addons:list u
+or
+
+    php eecli.php addons:list uninstalled
```

Comment:
@matthewjohns0n same here?  List add-ons that can be uninstalled?  VS ones that are currently uninstalled.

I'm thinking the past tense doesn't make sense for for an action that's currently being taken.

Thoughts?